homeassistant/components/zwave/__init__.py

```diff
@@ -266,9 +264,13 @@ def get_config_value(node, value_index, tries=5):
     from openzwave.network import ZWaveNetwork
     from openzwave.group import ZWaveGroup
 
-    config = {}
+    # Merge config entry and yaml config
+    config = config_entry.data
     if DATA_ZWAVE_CONFIG in hass.data:
-        config = hass.data[DATA_ZWAVE_CONFIG]
+        config.update(hass.data[DATA_ZWAVE_CONFIG])
```

Comment:
Let's make a copy so that we don't override the data in the config entry accidentally.

```python
config = {**config, **hass.data[DATA_ZWAVE_CONFIG]}
```